tools/oav-traffic-converter/ci.yml

```diff
@@ -0,0 +1,122 @@
+            mac:
+              imageName: 'macos-latest'
+            windows:
+              imageName: 'windows-latest'
+        pool:
+          vmImage: $(imageName)
+        steps:
+        - task: NodeTool@0
+          inputs:
+            versionSpec: '$(NodeVersion)'
+          displayName: 'Install Node.js'
+
+        - script: |
+            npm install
+          displayName: 'npm install'
+          workingDirectory: $(System.DefaultWorkingDirectory)/tools/oav-traffic-converter
+
+        - script: |
+            npm pack
+          displayName: 'npm pack'
+          workingDirectory: $(System.DefaultWorkingDirectory)/tools/oav-traffic-converter
+          condition: contains(variables['imageName'], 'ubuntu')
+
+        - script: 'cp oav-traffic-converter-*.tgz $(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
+          displayName: 'copy to staging dir'
+          workingDirectory: $(System.DefaultWorkingDirectory)/tools/oav-traffic-converter
+          condition: contains(variables['imageName'], 'ubuntu')
+
+        - task: PublishBuildArtifacts@1
+          inputs:
+            PathtoPublish: '$(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
+            ArtifactName: '$(VAR_ARIFACT_NAME)'
+            publishLocation: 'Container'
+          condition: contains(variables['imageName'], 'ubuntu')
+        
+  - ${{if ne(variables['Build.Reason'], 'PullRequest')}}:
+    - stage: Release
+      dependsOn: InstallAndTest
+      condition: succeeded()
+      jobs:
+      - job: approve
+        pool: server
+        steps:
+        - task: ManualValidation@0
+          inputs:
+            notifyUsers: 'Click to approve if it''s an expected public release.'
+      - job: release
+        dependsOn: approve
+        condition: and(succeeded(), ne(variables['USER_SKIP_PUBLIC_RELEASE'], 'true'))
+        steps:
+        - task: NodeTool@0
+          inputs:
+            versionSpec: '$(NodeVersion)'
+          displayName: 'Install Node.js'
+        - task: DownloadBuildArtifacts@0
+          inputs:
+            buildType: 'current'
+            downloadType: 'single'
+            artifactName: '$(VAR_ARIFACT_NAME)'
+            downloadPath: '$(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)'
+        - script: |
+            echo -e "\e[32m[$(date -u)] LOG: Ensure latest npm is installed"
+            sudo npm install -g npm || { echo 'npm install failed' ; exit 1; }
+            sudo npm install -g pnpm || { echo 'pnpm install failed' ; exit 1; }
+
+            echo -e "\e[32m[$(date -u)] LOG: publish the package"
+            echo "//registry.npmjs.org/:_authToken=$(azure-sdk-npm-token)" >> ~/.npmrc 
+            for file in $(VAR_BUILD_ARTIFACT_STAGING_DIRECTORY)/$(VAR_ARIFACT_NAME)/*.tgz
+            do
+              echo -e "\e[32m[$(date -u)] LOG: File: $file"
+              npm publish $file --access public || { echo 'publish $file failed' ; exit 1; }
+            done
+            rm ~/.npmrc || { echo 'rm ~/.npmrc failed' ; exit 1; }
```

Comment:
Use npm publish task from unified pipeline.

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/main/eng/tools/publish-to-npm.ps1

Reply:
@praveenkuttappan I thought this was internal to the tools repo. I'm going to combine the two conditional steps and just what's already there. I don't _currently_ have the time to move `publish-to-npm`. I need to publish this tonight.

Reply:
After I publish it, I'm ok with trying to help there :)